CreeDictionary/API/views.py

```diff
@@ -1,14 +1,28 @@
-from django.http import JsonResponse
+from typing import List, Union
 
-# todo: update api documentation
```

Comment:
The documentation was in a totally outdated Github wiki page, which I have just deleted. Now it's documented in `API/schema.py` as `SerializedSearchResult`